pkgs/build-support/bintools-wrapper/default.nix

```diff
@@ -60,6 +60,7 @@
 , postLinkSignHook ? null, signingUtils ? null
 }:
 
+assert propagateDoc -> bintools ? man;
```

Comment:
Is this one still necessary?  It's not explained in the commit message.

Reply:
It is not necessary but I found it helpful in debugging different issues so I left it for future hackers. I will document it.